integration_tests/test_aggregator_batch_commands.py

```diff
@@ -107,12 +107,11 @@ def on_market_cycle(self, market_info):
                 if transaction is None:
                     self.errors += 1
                 else:
-                    for response in transaction["responses"]:
-                        for area_response in response:
-                            if area_response['status'] == 'error':
+                    for area_uuid, response in transaction["responses"].items():
+                        for command_dict in response:
+                            if command_dict["status"] == "error":
```

Comment:
We're moving towards using single quotes for strings whenever possible (as of our style conventions). It would be great to have this aligned among all repos!